packages/active-campaign-client/src/handlers/deleteContact.ts

```diff
@@ -0,0 +1,36 @@
+import { APIGatewayProxyResult, SQSEvent } from 'aws-lambda';
+import { acClient } from '../activeCampaignClient';
+
+export async function handler(event: {
```

Comment:
If possible, you could distinguish handlers using different names:
```suggestion
export async function deleteContactHandler(event: {
```